assets/src/components/mapPage.tsx

```diff
@@ -101,13 +106,23 @@ const MapPage = (): ReactElement<HTMLDivElement> => {
           )}
         </div>
       </div>
+      <div className="m-map-page__map-container">
+        {selectedVehicle && showVehicleCard && (
+          <div className="m-map-page__vehicle-card">
+            <LiveVehicleCard
+              vehicle={selectedVehicle}
+              onClose={() => setShowVehicleCard(false)}
+            />
+          </div>
```

Comment:
One piece I'd particularly appreciate feedback on is having this VehicleCard as an element that is styled over the map, vs. passing it to the map as a control. Keeping it separate from the map seemed the lowest lift to implement, but it means that the card isn't visible when the map is put in fullscreen mode. I'm not sure that matters at this point, but can spend a little more time turning this into a map control if that seems worthwhile.

Reply:
Update: I ended up looking into it a bit more and looks like `LiveVehicleCard` could be passed as a child to the map container without needing explicitly be made into a control. That is fairly straightforward and would resolve the fullscreen issue described above. I'd still be interested in whether that seems like a sensible approach, or if there are benefits to keeping the `LiveVehicleCard` separate from the map component that I'm missing.

Reply:
I'd say that unless you discover additional downsides, passing it in as a child of the map would probably be best.